hack/cri-test/versions

```diff
@@ -0,0 +1,3 @@
+RUNC_VERSION=74a17296470088de3805e138d3d87c62e613dfc4
+CONTAINERD_VERSION=6c7abf7c76c1973d4fb4b0bad51691de84869a51
```

Comment:
RUNC_VERSION and CONTAINERD_VERSION are not used.

Reply:
yes，It's really not used

Reply:
I have removed unused variables